src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3641,4 +3641,60 @@ private double GetRelativeLuminance(System.Windows.Media.Color color)
             return 0.2126 * R + 0.7152 * G + 0.0722 * B;
         }
     }
+
+    /// <summary>
+    /// This converter is used to add margin/Padding from Popup in the CustomColorPicker
+    /// </summary>
+    public class AdditionConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if ((value != null) && (parameter != null))
+            {
+                var firstValue = System.Convert.ToDouble(value);
+                var secondValue = double.Parse(parameter as string);
+
+                return firstValue + secondValue;
+            }
+
+            return 0d;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
+        }
+    }
+
+    public class ColorToSolidColorBrushConverter : IValueConverter
+    {
+        /// <summary>
+        /// Converts a Color to a SolidColorBrush.
+        /// </summary>
+        /// <returns>
+        /// A converted SolidColorBrush. If the method returns null, the valid null value is used.
+        /// </returns>
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if (value != null)
+                return new SolidColorBrush((Color)value);
```

Comment:
So here we do not return `((SolidColorBrush)value).Color` property, like we do in `ConvertBack` below?
https://github.com/DynamoDS/Dynamo/pull/13794/files#diff-1eceea7bbe76f1c17266f4a4eafefa61ccf9c46412dc8c01ea9aa3c5aaef7a6cR3695
 Is that intended?

Reply:
not sure if I misunderstood your question but this is the explanation: the ColorToSolidColorBrushConverter converter converts from Color to SolidColorBrush and convert it back so using this SolidColorBrush((Color)value) will convert Color to SolidColorBrush.
For ConvertBack we receive a SolidColorBrush, so in order to convert it to Color we need to use ((SolidColorBrush)value).Color - first cast to SolidColorBrush and the get the SolidColorBrush.Color.
Let me know if I answered your comment.

Reply:
Yes, that answers it, thanks.